config_default.yaml

```diff
@@ -4,25 +4,25 @@ vagrant:
   # vagrant plugins are actually rubygems so you can declare to use any rubygems
   # common config level
   plugins:
+    # - _id: "essential-0"
+    #   name: vagrant-gatling-rsync
+    #   version: "" # empty means latest by default
+    #   state: uninstalled # uninstalled
+    #   enabled: false
+    #   config_key: gatling
+    #   options:
+    #     latency: 0.5
+    #     time_format: "%%H:%%M:%%S"
+    #     rsync_on_startup: true
+    # - _id: "essential-1"
+    #   name: vagrant-rsync-back
+    #   version: ""
+    #   state: uninstalled
+    #   enabled: false
     - _id: "essential-0"
-      name: vagrant-gatling-rsync
-      version: "" # empty means latest by default
-      state: installed # uninstalled
-      enabled: false
-      config_key: gatling
-      options:
-        latency: 0.5
-        time_format: "%%H:%%M:%%S"
-        rsync_on_startup: true
-    - _id: "essential-1"
-      name: vagrant-rsync-back
-      version: ""
-      state: installed
-      enabled: false
-    - _id: "essential-2"
       name: vagrant-hostmanager
       version: ""
-      state: installed
+      state: uninstalled
```

Comment:
I think it's better to set the default state as empty, it means:
- if users already install the plugin, it will be kept as it
- if the plugin is not installed, it will not be installed
=> expected behavior, the plugin config does nothing by default.

When we set the state as `uninstalled` (not expected behavior)
- if users already install the plugin, it will be uninstalled
- if the plugin is not installed, it will not be installed

Reply:
I fixed it! done!